# header, contains time stamp
Header header

# array of all perceived objects' motion
PerceivedObjectMotion[] perceived_objects